samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -79,7 +79,9 @@ public class StorageConfig extends MapConfig {
   public static final String STORE_RESTORE_FACTORIES = STORE_PREFIX + "%s." + RESTORE_FACTORIES_SUFFIX;
   public static final String JOB_RESTORE_FACTORIES = STORE_PREFIX + RESTORE_FACTORIES_SUFFIX;
   public static final List<String> DEFAULT_RESTORE_FACTORIES = ImmutableList.of(KAFKA_STATE_BACKEND_FACTORY);
+  public static final long DEFAULT_ROCKSDB_MAX_MANIFEST_FILE_SIZE_IN_BYTES = 1024 * 1024 * 1024L;
 
+  static final String DEFAULT_ROCKSDB_MAX_MANIFEST_FILE_SIZE = STORE_PREFIX + "default.rocksdb.max.manifest.file.size";
```

Comment:
I would prefer if we have a default store name variable.  
`static final String DEFAULT_STORE_NAME = "default";`
In the docs, mentioned that any config with the suffix `stores.default` prefix would reflect to all stores.